extensions/blocks/map/edit.js

```diff
@@ -68,6 +67,19 @@ class MapEdit extends Component {
 		};
 		this.mapRef = createRef();
 	}
+
+	componentDidUpdate( prevProps ) {
+		const { prevClassName } = prevProps;
+		const { className, setAttributes } = this.props;
+
+		if ( prevClassName !== className ) {
+			const mapStyleObject = settings.styles.find( styleObject =>
+				className.includes( styleObject.name )
```

Comment:
This is giving me an error: `TypeError: Cannot read property 'name' of undefined`.

I wonder if we should be using `import { getActiveStyleName } from '../../shared/block-styles';`